src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+        user.setAge(41);
+        assertThrows(CustomException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullAge_NotOk() {
+        User user = new User();
+        user.setAge(null);
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login,password or age is null";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void threeLettersLogin_NotOk() {
+        User user = new User();
+        user.setLogin("abc");
+        user.setPassword("abrakadabra");
+        user.setAge(83);
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login must be at least 6 characters long!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void fiveLettersLogin_NotOk() {
+        User user = new User();
+        user.setLogin("abcdf");
+        user.setPassword("messagetotheworld");
+        user.setAge(41);
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login must be at least 6 characters long!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void eightLettersLogin_Ok() {
+        User user = new User();
+        user.setLogin("Vitaliks");
+        user.setAge(22);
+        user.setPassword("normalpassword321");
+        registrationService.register(user);
+        assertTrue("Vitaliks".equals(user.getLogin()));
+    }
+
+    @Test
+    void emptyloginUser_NotOk() {
+        User user = new User();
+        user.setLogin("");
+        user.setAge(32);
+        user.setPassword("thebestpassword");
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login or password is empty";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void userOk_Ok() {
+        User user = new User();
+        user.setLogin("Misha4433");
+        user.setPassword("lutsk12345");
+        user.setAge(41);
+        registrationService.register(user);
+        assertTrue("Misha4433".equals(user.getLogin()));
+    }
+
+    @Test
+    void userLoginIsAlreadyExist_NotOk() {
+        User user = new User();
+        user.setLogin("Vitaliks");
+        user.setAge(62);
+        user.setPassword("great_password");
+        Exception exception = assertThrows(CustomException.class, () ->
+                registrationService.register(user));
+        String excepted = "User with this login is already register!";
```

Comment:
The variable name 'excepted' should be corrected to 'expected' for consistency with other test cases.